nodejs/sqlcommenter-nodejs/samples/express-opentelemetry/knexConnection.js

```diff
@@ -0,0 +1,29 @@
+const { wrapMainKnexAsMiddleware } = require("@google-cloud/sqlcommenter-knex");
+const Knex = require("knex");
+
+const sqlcommenterMiddleware = wrapMainKnexAsMiddleware(
+  Knex,
+  {
+    client_timezone: true,
+    db_driver: true,
+    route: true,
```

Comment:
This are supposed to be optional, let's add a disclaimer here that adding these fields isn't mandatory and will cause a high cardinality burst. Actually I'd start with firstly putting the most important fields `traceparent` and `tracestate` then these 3 below them after a newline, and with the comment.

Reply:
Ok will do. The actual code defaults should probably be updated, too:
https://github.com/google/sqlcommenter/blob/1bc1381d838c3f61f7a290ed757daf9cc2ef8870/nodejs/sqlcommenter-nodejs/packages/sqlcommenter-sequelize/index.js#L18-L22

Reply:
let me know how it looks now